packages/loot-design/src/components/modals/CreateLocalAccount.js

```diff
@@ -117,15 +117,31 @@ function CreateLocalAccount({ modalProps, actions, history }) {
                     justifyContent: 'flex-end',
                   }}
                 >
-                  <label style={{ userSelect: 'none' }}>
+                  <label style={{
+                    userSelect: 'none',
+                    textAlign: 'right'
+                  }}>
                     <input
                       name="offbudget"
                       type="checkbox"
                       checked={!!values.offbudget}
                       onChange={handleChange}
                       onBlur={handleBlur}
                     />{' '}
-                    Off-budget
+                    <span> Off-budget </span>
+                    <div style={{ fontSize: '0.7em', color: 'grey' }}>
+                      This cannot be changed later. <br /> See
+                      <a href="https://actualbudget.github.io/docs/Accounts/overview"
```

Comment:
Suggestion: link directly to the relevant section (although it does not make much of a difference)

```suggestion
                      <a href="https://actualbudget.github.io/docs/Accounts/overview/#off-budget-accounts"
```

I also think this should open in a new tab, you’ll need `target="_blank" rel="noopener noreferrrer"` to get that to happen.

Reply:
This should be fixed now!